test/fixtures/single-get.raml

```diff
@@ -1,8 +1,7 @@
 #%RAML 0.8
 
 title: World Music API
-baseUri: http://example.api.com/{version}
-version: v1
+baseUri: http://localhost:3333
```

Comment:
Why was this changed? Seems like ``single-get.raml`` was used by more than one test case. The abstract version is better than this one specific to our test environment setup. Maybe add another file ``single-get-localhost.raml`` for your test, but put the abstract one back like it was.